v21.2/security-reference/security-overview.md

```diff
@@ -114,7 +114,7 @@ Cockroach Labs maintains <a href="https://github.com/cockroachdb/cockroach">Cock
   <td>✓</td>
   <td>✓</td>
   <td>✓</td>
-  <td>Industry standard encryption-at-rest provided at the infrastructure level by your chosen infrastructure-as-a-service (IAAS) provider, either Google Cloud Platform (GCP) or Amazon Web Services (AWS). See documentation for <a href="https://cloud.google.com/compute/docs/disks#pd_encryption">GCP persistent disk encryption</a> or <a href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html">AWS Elastic Block Storage</a>.
+  <td>Industry-standard encryption-at-rest provided at the infrastructure level by your chosen deployment environment, either Google Cloud Platform (GCP) or Amazon Web Services (AWS). You can learn more about <a href="https://cloud.google.com/compute/docs/disks#pd_encryption">GCP persistent disk encryption</a> or <a href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html">AWS Elastic Block Storage</a>..
```

Comment:
If this description is about what could be available on SH, then Azure could be mentioned here. But if it's only about CC Dedicated's default encryption, then we could leave it as is.

Reply:
It includes self-hosted (see the relative path, cloud-specific stuff is in its own unversioned directory). I've added the Azure details in all instances.